docs/going-to-production.md

```diff
@@ -68,6 +68,13 @@ export async function getServerSideProps({ req, res }) {
   }
 }
 ```
+By default, `Cache-Control` headers will be set differently depending on how your page fetches data.
+
+If the page is using `getServerSideProps` or `getInitialProps`, then it will use the default `Cache-Control` header configured by `next start` in order to prevent accidental caching of responses that cannot be cached.
+
+If the page is using `getStaticProps` or automatic static optimization, then it will have s-maxage=REVALIDATE_SECONDS, stale-while-revalidate or if revalidate is not used s-maxage=31536000, stale-while-revalidate.
+
+If you want a different Cache-Control while using SSR you can use res.setHeader('Cache-Control', 'value_you_prefer').
```

Comment:
```suggestion
If the page is using `getServerSideProps` or `getInitialProps`, then it will use the default `Cache-Control` header configured by `next start` in order to prevent accidental caching of responses that cannot be cached. If you want a different cache behavior while using SSR you can use `res.setHeader('Cache-Control', 'value_you_prefer')`.

If the page is using `getStaticProps` or automatic static optimization, then it will have s-maxage=REVALIDATE_SECONDS, stale-while-revalidate or if revalidate is not used s-maxage=31536000, stale-while-revalidate.
```